packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/core",
-  "version": "2.2.9-alpha.4",
+  "version": "2.2.9-alpha.5",
```

Comment:
I'm not sure which version would be correct here.
`2.2.9-alpha.5` or `2.2.10-alpha.4`?